src/Features/Core/Portable/Completion/Providers/SymbolCompletionItem.cs

```diff
@@ -165,7 +186,12 @@ private static ISymbol DecodeSymbol(string id, Compilation compilation)
             var contextDocument = FindAppropriateDocumentForDescriptionContext(document, supportedPlatforms);
 
             var semanticModel = await contextDocument.GetSemanticModelAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
This seems odd: `FindAppropriateDocumentForDescriptionContext` looks in the `document.GetLinkedDocumentIds()` for supported platforms. The `semanticModel` is calculated with the linked document, but everything else later is calculated by using the original document (Line 168):
https://github.com/dotnet/roslyn/blob/dcc03e7d083c7950a4864d9343938047420624bf/src/Features/Core/Portable/Completion/Providers/SymbolCompletionItem.cs#L164-L171

This has been there before, so I don't know whether that is intentional (or does not matter). I just stumbled upon it.

Reply:
I don't know what method this is.  Are you fully sync'ed with roslyn?

Reply:
No. I'm on VS 16.7. and can not use a preview of VS. Therefore I can not merge master at the moment. We can postpone this discussion until the release of VS 16.8

Reply:
> No. I'm on VS 16.7. and can not use a preview of VS

Out of curiosity, why not?  They all rnu side-by-side.  For example, i routinely run any of these three versions depending on what i'm doing/testing:

![image](https://user-images.githubusercontent.com/4564579/97888728-550b7b80-1ce0-11eb-9ca8-08c22a893d1d.png)

Reply:
Corporate policy. I'm not allowed to install preview stuff.